src/View/Intro.elm

```diff
@@ -4,20 +4,46 @@ import Copy.Keys exposing (Key(..))
 import Copy.Text exposing (t)
 import Html exposing (..)
 import Html.Attributes exposing (..)
+import Html.Events exposing (..)
 import Message exposing (Msg(..))
 import Route exposing (Route(..))
 import Video exposing (Video(..), videoToData)
 import View.Video
 
 
-view : Html Msg
-view =
+view : Video -> Html Msg
+view activeVideo =
     div [ class "intro container-fluid" ]
         [ div [ class "row my-4" ]
             [ h1 [ class "d-block mx-auto px-4 text-center" ] [ text (t SiteTitle) ]
             ]
-        , div [ class "intro-video" ] [ View.Video.view (videoToData Intro1) ]
+        , div [ class "intro-video" ] [ View.Video.view (videoToData activeVideo) ]
+        , renderVideoButtons
         , div [ class "row my-4" ]
-            [ a [ class "btn btn-primary btn-lg mx-auto", href (Route.toString Route.Desktop) ] [ text (t StartNewGame) ]
+            [ a
+                [ class "btn btn-primary btn-lg mx-auto"
+                , href (Route.toString Route.Desktop)
+                ]
+                [ text (t StartNewGame) ]
             ]
         ]
+
+
+renderVideoButtons : Html Msg
+renderVideoButtons =
+    ol [ class "row my-4 list-unstyled justify-content-md-center" ]
```

Comment:
Probably want `max-width: 800px` on here to match the video above. I would usually put `display: flex` on the list and switch from `flex-direction: row` on desktop to `flex-direct: column` on mobile.

Reply:
Is there a good bootstrap way to do that?